pom.xml

```diff
@@ -94,8 +94,7 @@ under the License.
 
   <properties>
     <javaVersion>11</javaVersion>
-    <maven.compiler.source>${javaVersion}</maven.compiler.source>
-    <maven.compiler.target>${javaVersion}</maven.compiler.target>
+    <maven.compiler.release>${javaVersion}</maven.compiler.release>
```

Comment:
Bah, forgot that this should not be removed :smile: 

Due this in parent:
https://github.com/apache/maven-parent/blob/maven-parent-41/pom.xml#L956-L957

Hence 11 becomes "1.11", which is kinda fine so far (as javac will ignore those two when release set), but reporting craps out, so release becomes impossible. Will undo this.

Reply:
oh no!

Reply:
Fixed with cbf1b5bcd406c57c24796c3561e3036fe651f27e